sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/MetricData.java

```diff
@@ -29,6 +29,8 @@
           /* isMonotonic= */ false, AggregationTemporality.CUMULATIVE, Collections.emptyList());
   private static final DoubleSummaryData DEFAULT_DOUBLE_SUMMARY_DATA =
       DoubleSummaryData.create(Collections.emptyList());
+  private static final DoubleHistogramData DEFAULT_DOUBLE_HISTOGRAM_DATA =
+      DoubleHistogramData.create(AggregationTemporality.CUMULATIVE, Collections.emptyList());
```

Comment:
can we really hard-code CUMULATIVE here? Shouldn't it depend on how the aggregation is configured?

Reply:
it's a default & fallback value when we try to get the histogram data from an instance of `MetricData` whose type is not `HISTOGRAM`. I would assume something is wrong if this happens.